### ISSUE TITLE
Provide element details info for MMS elements

### ISSUE THREAD

**Issue by bergmanngabor** [IncQueryLabs/incquery-server-jupyter#8]

**Comment by istvanrath** [comment]
* use https://mms.openmbee.org/alfresco/mms/swagger-ui/index.html
* investigate https://pypi.org/project/mms-python-client/

**Comment by bergmanngabor** [comment]
@istvanrath Are we happy with the current status of this feature?
(available for triage on the `develop` branch and the corresponding binder)

**Comment by istvanrath** [comment]
I have two remarks:

1) Even if we don't split MMS features in terms of namespace etc, I think it would be better to offer MMS-related functionality on a _runtime stand-alone_ basis, i.e. without requiring to instantiate and initialize IQS-related features.

2) For the sake of completeness, we should still aim for a (minimal) integration of the features offered by https://pypi.org/project/mms-python-client/, e.g. publish parts of the report back to VE:


```
import mms_python_client

class Adapter:

    def __init__(self, client, server):
        self.client = client
        self.client.configuration.host = server

    def login(self, username, password):
        self.client.configuration.username = username
        self.client.configuration.password = password
        del username, password

class MMSAdapter(Adapter):

    def __init__(self, server, project_id, ref_id):
        super().__init__(mms_python_client.ApiClient(), server)
        self.project_id = project_id
        self.ref_id = ref_id
        self.elem_api = mms_python_client.api.ElementApi(self.client)
        self.art_api = mms_python_client.api.ArtifactApi(self.client)

    def update_documentation(self, element_id, documentation):
        target = {"id": element_id, "documentation": documentation}
        payload = {"elements": [target]}
        return self.elem_api.post_elements(self.project_id, self.ref_id, payload)

    def publish_image(self, element_id, image):
        artifact_response = self.art_api.post_artifact(self.project_id, self.ref_id, element_id, image)
        # TODO: this is hardcoded
        artifact_url = "https://opencae.jpl.nasa.gov/alfresco" + artifact_response.artifacts[0].artifact_location
        return self.update_documentation(element_id, "<img src=\"" + artifact_url + "\" />")

    def get_element(self, element_id):
        return self.elem_api.get_element(self.project_id, self.ref_id, element_id)
```

**Comment by bergmanngabor** [comment]
As for (1), this is solved in commit 187ef07. 

As for (2), well, you can already use the `mms` object (of type `MMSClient`) in the notebook to use any of the MMS APIs, similarly to the `iqs` object. For instance, use `mms.artifact.post_artifact()` to upload an image. Although the exact workflow hardcoded in the above code sample is not included in the notebook (yet?), and I am unsure whether it would be a wise idea in a public demo.

**Comment by abelhegedus** [comment]
This seems to be done based on comments and commits, so closing.
